validator-auction-static/js/chart.js

```diff
@@ -221,7 +211,7 @@ function renderChart(bids, priceFunction, currentBlocktimeInMs, remainingSeconds
     Chart.defaults.global.defaultFontSize = 16
 }
 
-function getAuctionData() {
+function getAuctionData(duration = 800) {
```

Comment:
We should rename this. I did not know what duration is nor did I know that this will render something.
What about `fetchAuctionDataAndRender(options={animationDuration: 800})`